cmd/mungedocs/links_test.go

```diff
@@ -45,10 +45,6 @@ func TestGoodLinks(t *testing.T) {
 		expected string
 	}{
 		{"", ""},
-		{"[README](https://github.com/kubernetes/kubernetes/tree/master/README.md)",
```

Comment:
These are not broken but would require me to export the root README which seems kind of dumb since we test the same thing with other READMEs. I don't want to polute the root BUILD file for a duplicative test of a binary that we don't even run anymore.